mkdocs.yaml

```diff
@@ -16,6 +16,7 @@ nav:
           - Continuous Integration: odk-workflows/ContinuousIntegration.md
           - Your ODK Repository Overview: odk-workflows/RepositoryFileStructure.md 
       - Contributing: contributing.md
+  - Statistics: statistics.md
```

Comment:
1. How do you like a page called "Statistics"? Or should I just make a page called "Progress report"?
2. Where do you want the page? Here in the docs sounds like a good place. Do you want "Statistics" to be a major sidebar section? Or should I nest under something else?